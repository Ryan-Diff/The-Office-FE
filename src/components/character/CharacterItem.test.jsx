import React from 'react';
import { render, cleanup } from '@testing-library/react';
import CharacterItem from './CharacterItem';

describe('CharacterItem component', () => {
    afterEach(() => cleanup());

    it('renders a single character', () => {
        const { asFragment } = render(
            <CharacterItem
                name='Dwight Schrute'
                imageUrl='https://upload.wikimedia.org/wikipedia/en/c/cd/Dwight_Schrute.jpg'
            />);
        expect(asFragment()).toMatchSnapshot();
    });
});