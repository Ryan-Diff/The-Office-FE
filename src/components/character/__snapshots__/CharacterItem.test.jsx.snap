// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterItem component renders a single character 1`] = `
<DocumentFragment>
  <figure>
    <img
      alt="Dwight Schrute"
      src="https://upload.wikimedia.org/wikipedia/en/c/cd/Dwight_Schrute.jpg"
    />
    <figcaption>
      Dwight Schrute
    </figcaption>
  </figure>
</DocumentFragment>
`;
